### ISSUE TITLE
Accessibility on fresh Ubuntu

### ISSUE THREAD

**Issue by guyster104** [libretro/RetroArch#17018]
I cannot get accessibility started on a fresh install of Ubuntu 24.04. I have the dependencies for Orca and Espeak installed, with apt build-dep orca espeak
From the command line, I typed retroarch --accessibility
I expect to hear retroarch accessibility on, but hear nothing.
Retroarch is version 1.19.1, installed from the libretro repository.
Orca is the version that comes with Ubuntu-mate 24.04 by default

Bug can be reproduced easily by freshly installing Ubuntu-mate 24.04, adding the libretro repository, installing retroarch, along with the dependencies for Orca and Espeak
Architecture used is 64-bit.

.# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

[Description of the bug]

### Expected behavior

[What you expected to happen]

### Actual behavior

[What is actually happening]

### Steps to reproduce the bug

1. [First step]
2. [Second step]
3. [and so on...]

### Bisect Results

[Try to bisect and tell us when this started happening]

### Version/Commit
You can find this information under Information/System Information

- RetroArch: [version/commit]

### Environment information

- OS: [The operating system you're running]
- Compiler: [In case you are running local builds]

**Comment by guyster104** [comment]
Never mind. I would have thought Espeak was installed, since Orca uses it to speak. Apparently not. sudo apt install espeak did the trick.
